.github/workflows/image-push.yml

```diff
@@ -0,0 +1,46 @@
+name: Build And Push
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  push-multiarch:
+    name: Build and Push amd64 and arm64
```

Comment:
I think `permissions:` should be added here as well, I think we should be using one of the merged repos as reference: 

- https://github.com/rancher/image-build-coredns/blob/master/.github/workflows/image-push.yml
OR
https://github.com/rancher/image-build-flannel/blob/master/.github/workflows/image-push.yml